# William R Spitzer <hololeap@users.noreply.github.com> (2021-05-19)
# 
# This patch adds a needed import to one of the test files. It also adds
# hspec as a dependency. tasty-hspec-1.1.7 no longer re-exports Test.Hspec
# See <https://hackage.haskell.org/package/tasty-hspec-1.1.7/changelog>
# 
# From: William R Spitzer <hololeap@users.noreply.github.com>
# Signed-off-by: William R Spitzer <hololeap@users.noreply.github.com>

diff -urN monad-dijkstra-0.1.1.3/monad-dijkstra.cabal monad-dijkstra-0.1.1.3-new/monad-dijkstra.cabal
--- monad-dijkstra-0.1.1.3/monad-dijkstra.cabal	2001-09-08 19:46:40.000000000 -0600
+++ monad-dijkstra-0.1.1.3-new/monad-dijkstra.cabal	2021-05-18 18:45:05.422827949 -0600
@@ -32,6 +32,7 @@
                      , tasty
                      , tasty-hspec
                      , monad-dijkstra
+                     , hspec
   default-language:    Haskell2010
   ghc-options:         -Wall -threaded -rtsopts -with-rtsopts=-N
 
diff -urN monad-dijkstra-0.1.1.3/test/Main.hs monad-dijkstra-0.1.1.3-new/test/Main.hs
--- monad-dijkstra-0.1.1.3/test/Main.hs	2001-09-08 19:46:40.000000000 -0600
+++ monad-dijkstra-0.1.1.3-new/test/Main.hs	2021-05-18 18:00:55.994812724 -0600
@@ -9,6 +9,7 @@
 
 import           Test.Tasty
 import           Test.Tasty.Hspec
+import           Test.Hspec
 
 data Side = L | R
     deriving (Eq, Show)
